ueditor导入导出word

risc64 opened the pull request:
ueditor能否导入导出word带文本格式？包括字体、居中，分页等

sjshare committed:
修改image.js本地图片上传

修改本地图片上传，当上传文件出现超出大小或格式不允许时，移除文件后文件总数及总大小出错问题。

sjshare committed:
修改attachment.js本地文件上传

修改本地文件上传，当上传文件超出大小或格式不允许时，点击移除后文件总数及总大小出错bug。

Dafrok committed:
Merge pull request #2826 from fex-team/dev-2.0.0

更改gruntfile和package.json中过时的配置项

Dafrok committed:
修改package.json 增加uglify的dependence

tangtanglove committed:
Update Uploader.class.php

iconv函数无法将unicode转换为utf-8（会有警告），更换为mb_convert_encoding函数

Phinome committed:
Update ISSUE_TEMPLATE.md

增加声明

Phinome committed:
add FAQ wiki

Phinome committed:
Merge pull request #2781 from sjshare/dev-1.5.0

修改本地文件上传bug

Phinome committed:
Merge pull request #2869 from tangtanglove/patch-1

Update Uploader.class.php

Phinome committed:
Update README.md

lzxb committed:
Update buttonicon.css

修复在webpack下的bug：
Module not found: Error: Cannot resolve 'file' or 'directory' ../images/charts.png  in

Phinome committed:
Merge pull request #3201 from lzxb/dev-1.5.0

Update buttonicon.css

houyhea committed:
修改htmlParser的正则表达式,增强html解析的容错能力

现有的htmlParser 节点正则表达式无法匹配:<div style="text-align:center;font-family:" font-size:14px;"=""><img src="http://hs-album.oss.aliyuncs.com/static/27/78/35/image/20161206/20161206174331_41105.gif" alt="" /><br /></div>
修改正则表达式:

Phinome committed:
Merge pull request #3205 from houyhea/dev-1.5.0

修改htmlParser的正则表达式,增强html解析的容错能力

Commit:
Fixed autoheight Plugin onscroll error

修复 autoheight 插件在 UE 实例销毁后，因 onscroll 事件监听没有正常取消监听导致 me.window 报 undefined 错误的问题

woleicom committed:
Create image.js

更改图片尺寸大小bug

Phinome committed:
Merge pull request #3219 from woleicom/dev-1.5.0

Create image.js

Phinome committed:
Merge pull request #3216 from jinzhk/patch-1

修复 autoheight 插件在 destroy 后 onscroll 报错的问题

Phinome committed:
prettier format file

Commit:
注释掉关于表头合并的特殊处理逻辑

regrexd committed:
update issue template

regrexd committed:
update comment

Phinome committed:
Merge pull request #3221 from regrex/dev-1.5.0

干掉关于table表头合并逻辑的特殊处理，该逻辑会导致表头合并出错

Commit:
解决执行 grunt defalut 命令时抛出错误 TypeError: Cannot read property '1' of null

Phinome committed:
Merge pull request #3223 from jinhuiWong/dev-1.5.0

解决执行 grunt defalut 命令时抛出错误 TypeError: Cannot read property '1' of null

Commit:
优化远 程抓取图片插件(catchremoteimage) 的相关代码：
1.支持背景图片的抓取
2.抓取图片时添加loading
3.抓取失败时显示失败的图片而不是不处理
4.事件 catchremotesuccess 添加成功抓取和抓取失败的dom列表参数

Phinome committed:
Merge pull request #3224 from jinhuiWong/dev-1.5.0

优化远 程抓取图片插件(catchremoteimage) 的相关代码

Commit:
fix bug
抓取远程图片失败后 _src 应是failIMG , 不然会影响getContent()输出的值

Phinome committed:
Merge pull request #3228 from jinhuiWong/dev-1.5.0

fix bug

Commit:
添加基于内容的判断方式

Commit:
添加基于内容的判断方式

txb2d commented:
我用的是php版本的ueditor，多图上传和远程抓图，每次上传一张图片并入库时候，我想往客户端写一个cookie。这个是做附件关联文章使用。可是郁闷的事情发上了，多图上传为什么，只把最后一个入库的图片或者附件ID写入cookie。前面的上传入库的图片或者附件id。都不能写入cookie~怎么回事的好几天了，我都没有解决这个问题！谢谢！！

Phinome committed:
新增安全通告

新增安全通告

jianzhiyao committed:
修复当图片带有GET参数时，无法本地化的bug

例如：
http://img.xiumi.us/xmi/ua/cq5i/i/a662444d6c6b9fe35c5ca77666209612-sz_18655.png?x-oss-process=image/resize,w_640/auto-orient,1/crop,x_58,y_0,w_435,h_139

jianzhiyao committed:
使用PHP自带方法分析url，并拼接还原为原本的带GET请求url

Commit:
修复调整单元格对齐方式会导致textarea元素丢失的问题

修复问题：在关闭xss过滤前提下，当选择多个td（td中含有textarea）并修改单元格对齐方式时，td中的textarea会被清空

conwnet committed:
修正大小写问题

小写的 timezone_set Asia/chongqing 在 linux 下有问题。

Fenzland committed:
修复编辑源码状态下，`.focus()`和`.blur()`失效的问题

## 问题
编缉源码状态下，`.blur()` 方法无法使编辑器失去焦点；`.focus()` 方法不但无法使编缉器获得焦点，还会使编辑器无法接收键盘操作。

## 解决
修改 source 插件，使用与 `.getContent()` 和 `.setContent()` 相同的处理方式，在开启源码编辑模式时对编辑器的 `.focus()` 和 `.blur()` 方法进行覆盖，并在关闭源码编辑模式时还原。

Phinome committed:
Merge pull request #3238 from Fenzland/fix-focus&blur-in-source-mode

修复编辑源码状态下，`.focus()`和`.blur()`失效的问题

Phinome committed:
Merge pull request #3237 from conwnet/patch-2

修正大小写问题

Phinome committed:
Merge pull request #3234 from jianzhiyao/patch-2

修复当图片带有GET参数时，无法本地化的bug

Phinome committed:
Merge pull request #3236 from tylanbin/patch-2

修复调整单元格对齐方式会导致td中textarea元素丢失的问题

Phinome committed:
upgrade apache commons-fileupload jar to fixe old versions bug, causing ddos

Phinome committed:
add apache common-fileupload jar pacakge security alarm

Mrshihe commented:
ueditor导出成word有办法了么？

lujiaxing committed:
Update image.js

Make the header same with /plugins/autoupload.js
See:
/fex-team/ueditor/blob/089c7dee81cf06bf68a4ff6a362a7dcf4511d78b/_src/plugins/autoupload.js#L119

Phinome committed:
Merge pull request #3247 from lujiaxing/patch-1

Update image.js to make image uploader send same header with drag-drop-uploading

Commit:
.net 后端示例代码有安全风险，暂时删除

Tuyuda commented:
> ueditor导出成word有办法了么？

JS导出吧，兄弟

wonderzhou committed:
更新自定义插件开发相关文档说明

Phinome committed:
Merge pull request #3250 from wonderzhou/dev-1.5.0

更新二次开发自定义插件相关文档说明

nik2011 commented:
> ueditor能否导入导出word带文本格式？包括字体、居中，分页等

请问 UEditorSnapscreen插件为什么没用效果

Commit:
后端代码只是 demo，有太多安全漏洞，为了避免误用，将不再提供

chenpanpan0809 commented:
现在ueditor能不能使用word导入预览，保存，并导出

Commit:
Update README.md

Commit:
Merge pull request #3260 from nwind/patch-2

Update README.md
